roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+  - name: add nginx key
+    apt_key:
+      id: '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62'
+      data: "{{ lookup('file', 'nginx.gpg') }}"
+      state: present
```

Comment:
Could this be done like this?
```yaml
- name: add nginx key
  apt_key:
    url: https://nginx.org/keys/nginx_signing.key
```
so it will always grab whatever the latest key is?